internal/planner/planner_test.go

```diff
@@ -19,13 +21,12 @@ func TestPlannerHelloWorld(t *testing.T) {
 	// test/wasm/ directory that are specified in YAML, compiled into Wasm, and
 	// executed inside of a node program. For the time being, the planner is
 	// simple enough that exhaustive unit testing is not as valuable as
-	// end-to-end testing. These tests provide a quick sanity check that the
+	// end-to-end testing. These tests provide a quick consistency check that the
```

Comment:
Thanks for changing this comment here and below! Sanity check isn't very clear and is somewhat ableist. 👍

Reply:
Yeah, I've taken this opportunity to sneaked this in. 😄